SparkPost/2web2ui
# SD-1263 - New warmup style when editing IP pools

[SD-1263](https://jira.int.messagesystems.com/browse/SD-1263)

### What Changed
- Updated `<IpForm/>` component to meet new requirements
- Moved some component rendering logic in the `<EditIpPage/>` component inside the return statement
- Removed space above `.CardContainer` component instances to make the component more re-usable
- Updated styles for `<Card/>` component to use standard units of spacing
- Updated styles for the new IP form - NOTE: These are all great examples that would better be handled with something like [Styled System](https://styled-system.com/)

#### Before:

![Screen Shot 2019-11-25 at 10 09 48 AM](https://user-images.githubusercontent.com/3613392/69552054-c8066100-0f6b-11ea-9056-c044350b064c.png)

#### After:

![Screen Shot 2019-11-25 at 10 09 31 AM](https://user-images.githubusercontent.com/3613392/69552061-cc327e80-0f6b-11ea-83b9-8ccab47183cf.png)

### How To Test
- using `appteam`, go to: `/account/ip-pools/edit/carnold_all_cold/35.160.52.243`
- Verify that the warmup stage `<select>` only renders when Auto Warmup is enabled
- Verify that the warmup stage `<select>` does not render when Auto Warmup is disabled
- Reassign the IP pool
- Turn off IP warmup
- Verify that changes were appropriately saved and that the IP pool was successfully re-assigned
- Verify that 'Engagement Based IP Warmup' is not a selectable option, but rather, a note to ask a TAM about the feature
- Verify that links navigate to the appropriate pages in the documentation
- Verify that requests (with relevant data passed) are made to the relevant APIs as they do in the current implementation

### To Do
- [x] Still verifying URLs for links
